build/package/LICENSE

```diff
@@ -1788,3 +1788,370 @@ licenses.
     software is granted. This software is provided "as is" without
     express or implied warranty, and with no claim as to its
     suitability for any purpose.
+
+- @protobufjs/aspromise@<VERSION>
+- @protobufjs/base64@<VERSION>
+- @protobufjs/codegen@<VERSION>
+- @protobufjs/eventemitter@<VERSION>
+- @protobufjs/fetch@<VERSION>
+- @protobufjs/float@<VERSION>
+- @protobufjs/inquire@<VERSION>
+- @protobufjs/path@<VERSION>
+- @protobufjs/pool@<VERSION>
+- @protobufjs/utf8@<VERSION>
+- protobufjs@<VERSION>
```

Comment:
For all the existing dependencies we only list the `extension/dist/ext/extension.js` file and then say "`his product bundles <product name>`. Since all these dependencies just end up getting merged into extension.js. I'm not sure it's necessary to individually list out all the npm packages but instead just list the product name.

Reply:
Yeah I am thinking we need to fix that as well because `extension/dist/ext/extension.js` isn't a file, I believe its actually `dist/ext/extension.js`

Reply:
Since this LICENSE is for the .visx file, the path refers to the `extension.js` file in the .vsix, which I believe is in `extension/dist/ext/extension.js`.

Reply:
Oh yeah, yeah my bad inside of the VSIX it is under that extension folder